worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -424,6 +421,20 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
     expiredShuffleKeys.asScala.foreach { shuffleKey =>
       logInfo(s"Cleanup expired shuffle $shuffleKey.")
       if (fileInfos.containsKey(shuffleKey)) {
+        fileInfos.get(shuffleKey).asScala.foreach {
```

Comment:
```val shuffleFiles = fileInfos.remove(shuffleKey)``` to reuse

Reply:
now use shuffleFiles  to traverse the fileinfos to delete all the expired writers